### ISSUE TITLE
Bundle fallback SSH client on Windows

### ISSUE THREAD

**Issue by techtonik** [mitchellh/vagrant#4884]
Vagrant doesn't work out of the box on Windows. This can be changed if it can bundle fallback ssh client, e.g. if main client is not found, attempt to execute one from its own directory. Currently, I do it like this (which is not quite good, because it adds too many unix commands to PATH):

```
E:\p\gratipay.com>vagrant ssh
`ssh` executable not found in any directories in the %PATH% variable. Is an
SSH client installed? Try installing Cygwin, MinGW or Git, all of which
contain an SSH client. Or use your favorite SSH client with the following
authentication information shown below:

Host: 127.0.0.1
Port: 2222
Username: vagrant
Private key: C:/Users/user/.vagrant.d/insecure_private_key

E:\p\gratipay.com>set PATH=%PATH%;C:\Program Files\Git\bin
```

**Comment by techtonik** [comment]
@sethvargo from here - https://www.vagrantup.com/downloads.html

**Comment by techtonik** [comment]
4th most popular problem - FWIW - https://stackoverflow.com/questions/tagged/vagrant

**Comment by tknerr** [comment]
@sethvargo were you wondering whether it was installed as a gem vs. the installer?

All I can say is that the windows installer does not ship with an ssh and / or rsync executable on the PATH. If have them from somewhere else on the PATH that's fine, if not you simply can not use `vagrant ssh` (or any rsync based provisioning / shared foldering) on Windows.

@techtonik that's why I do [this](https://github.com/tknerr/bills-kitchen/tree/master/files/tools)

**Comment by techtonik** [comment]
I install Vagrant from Windows .msi

**Comment by raphinesse** [comment]
:+1:  I would however suggest that vagrant uses the bundled client by default. Overriding should of course be possible via some configuration option.

**Comment by chrisroberts** [comment]
Fixed via mitchellh/vagrant-installers#107
